src/js/stores/DCOSStore.js

```diff
@@ -26,50 +27,42 @@ class DCOSStore extends EventEmitter {
 
     this.data = {
       marathon: new ServiceTree(),
-      mesos: {
-        frameworks: []
-      },
+      mesos: new SummaryList(),
       dataProcessed: false
     };
 
     this.proxyListeners = [
       {
         event: MARATHON_GROUPS_CHANGE,
-        handler: this.processMarathonData,
+        handler: this.onMarathonGroupsChange,
         store: MarathonStore
       },
       {
         event: MESOS_SUMMARY_CHANGE,
-        handler: this.processMesosData,
+        handler: this.onMesosSummaryChange,
         store: MesosSummaryStore
       }
     ];
   }
 
-  /**
-   * Process Marathon data
-   * @param {ServiceTree} data
-   */
-  processMarathonData(data) {
-    if (!(data instanceof ServiceTree)) {
+  onMarathonGroupsChange() {
+    let groups = MarathonStore.get('groups');
+    if (!(groups instanceof ServiceTree)) {
       return;
     }
 
-    this.data.marathon = data;
+    this.data.marathon = groups;
```

Comment:
i love :heart: love :heart: love :heart: love :heart: love :heart: love :heart: love :heart: love :heart: love :heart: the fact that this store doesn't use the getsetmixin